aiida/transport/plugins/test_local.py

```diff
@@ -48,6 +48,17 @@ def test_basic(self):
         with LocalTransport():
             pass
 
+    def test_is_open(self):
```

Comment:
This test should probably be defined in `test_all_plugins.py` with the `@run_for_all_plugins` decorator?